support tailwind 3.3.0

we noticed when tailwind moved forward past 3.2.7_postcss@8.4.21 it breaks because `tailwindcss/stubs/tailwindConfig.stub.js` was missing or something
```
Error: Cannot find module 'tailwindcss/stubs/defaultConfig.stub.js'
Require stack:
- /Users/zhenya/scratch/node_modules/.pnpm/@dxos+react-components@0.1.32-next.77d513e_slswrxsjzrgpkm6tkrz7tq2vui/node_modules/@dxos/react-components/dist/plugin/node/plugin.cjs
- /Users/zhenya/scratch/node_modules/.pnpm/@dxos+react-components@0.1.32-next.77d513e_slswrxsjzrgpkm6tkrz7tq2vui/node_modules/@dxos/react-components/plugin.js
- /Users/zhenya/scratch/vite.config.ts
- /Users/zhenya/scratch/node_modules/.pnpm/vite@4.2.1_@types+node@18.15.10/node_modules/vite/dist/node/chunks/dep-79892de8.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/zhenya/scratch/node_modules/.pnpm/@dxos+react-components@0.1.32-next.77d513e_slswrxsjzrgpkm6tkrz7tq2vui/node_modules/@dxos/react-components/dist/plugin/node/plugin.cjs:46:41)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at _require.extensions.<computed> [as .js] (file:///Users/zhenya/scratch/node_modules/.pnpm/vite@4.2.1_@types+node@18.15.10/node_modules/vite/dist/node/chunks/dep-79892de8.js:63989:17)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
 ELIFECYCLE  Command failed with exit code 1.
```

--- a/packages/apps/quickstart/package.json
+++ b/packages/apps/quickstart/package.json
@@ -42,7 +42,7 @@
     "@vitejs/plugin-react": "^3.0.1",
     "autoprefixer": "^10.4.12",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-pwa": "^0.14.1",

--- a/packages/apps/tasks-app-2/package.json
+++ b/packages/apps/tasks-app-2/package.json
@@ -43,7 +43,7 @@
     "@vitejs/plugin-react": "^3.0.1",
     "autoprefixer": "^10.4.12",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9"
   }

--- a/packages/apps/templates/hello-template/package.json
+++ b/packages/apps/templates/hello-template/package.json
@@ -19,7 +19,7 @@
     "@dxos/bare-template": "workspace:*",
     "@dxos/plate": "workspace:*",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7"
+    "tailwindcss": "~3.3.2"
   },
   "devDependencies": {
     "typescript": "^5.0.4"

--- a/packages/devtools/devtools-extension/package.json
+++ b/packages/devtools/devtools-extension/package.json
@@ -59,7 +59,7 @@
     "postcss": "^8.4.21",
     "rmdir": "~1.2.0",
     "safe-compare": "~1.1.4",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0"
   }

--- a/packages/devtools/devtools/package.json
+++ b/packages/devtools/devtools/package.json
@@ -63,7 +63,7 @@
     "@types/react-vis": "^1.11.11",
     "@vitejs/plugin-react": "^3.0.1",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/experimental/kai-frames/package.json
+++ b/packages/experimental/kai-frames/package.json
@@ -107,7 +107,7 @@
     "faker": "^5.5.3",
     "monaco-editor": "^0.25.0",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/experimental/kai-framework/package.json
+++ b/packages/experimental/kai-framework/package.json
@@ -97,7 +97,7 @@
     "postcss": "^8.4.21",
     "react-router": "^6.4.0",
     "storybook-addon-react-router-v6": "^0.3.2",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/experimental/kai/package.json
+++ b/packages/experimental/kai/package.json
@@ -65,7 +65,7 @@
     "postcss": "^8.4.21",
     "react-router": "^6.4.0",
     "storybook-addon-react-router-v6": "^0.3.2",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/experimental/kube-console/package.json
+++ b/packages/experimental/kube-console/package.json
@@ -85,7 +85,7 @@
     "@types/three": "^0.149.0",
     "@vitejs/plugin-react": "^3.0.1",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/experimental/mosaic/package.json
+++ b/packages/experimental/mosaic/package.json
@@ -48,7 +48,7 @@
     "@vitejs/plugin-react": "^3.0.1",
     "faker": "^5.5.3",
     "postcss": "^8.4.21",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/experimental/plexus/package.json
+++ b/packages/experimental/plexus/package.json
@@ -41,7 +41,7 @@
     "faker": "^5.5.3",
     "postcss": "^8.4.21",
     "string-hash": "~1.1.3",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "typescript": "^5.0.4",
     "vite": "^4.3.9",
     "vite-plugin-fonts": "^0.7.0",

--- a/packages/ui/aurora-theme/package.json
+++ b/packages/ui/aurora-theme/package.json
@@ -26,7 +26,7 @@
     "lodash.merge": "^4.6.2",
     "postcss": "^8.4.21",
     "tailwind-merge": "^1.10.0",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "tailwindcss-logical": "^3.0.0",
     "tailwindcss-radix": "^2.8.0"
   },

--- a/packages/ui/aurora-theme/src/config/tailwind.ts
+++ b/packages/ui/aurora-theme/src/config/tailwind.ts
@@ -13,7 +13,7 @@ import merge from 'lodash.merge';
 import tailwindcssLogical from 'tailwindcss-logical';
 import tailwindcssRadix from 'tailwindcss-radix';
 import tailwindColors from 'tailwindcss/colors';
-import defaultConfig from 'tailwindcss/stubs/defaultConfig.stub.js';
+import defaultConfig from 'tailwindcss/stubs/config.full.js';
 import { Config, ThemeConfig } from 'tailwindcss/types/config';
 
 export type TailwindConfig = Config;

--- a/packages/ui/aurora-theme/src/typings.d.ts
+++ b/packages/ui/aurora-theme/src/typings.d.ts
@@ -2,6 +2,6 @@
 // Copyright 2022 DXOS.org
 //
 
-declare module 'tailwindcss/stubs/defaultConfig.stub.js';
+declare module 'tailwindcss/stubs/config.full.js';
 declare module 'tailwindcss-radix';
 declare module 'tailwindcss-logical';

--- a/tools/presenter/package.json
+++ b/tools/presenter/package.json
@@ -53,7 +53,7 @@
     "remark-parse-frontmatter": "^1.0.3",
     "remark-unwrap-texts": "^1.0.3",
     "rollup": "^3.20.4",
-    "tailwindcss": "~3.2.7",
+    "tailwindcss": "~3.3.2",
     "tailwindcss-logical": "^3.0.0",
     "tailwindcss-radix": "^2.8.0",
     "typescript": "^5.0.4",

